dev-support/jenkins-scripts/generate-hbase-website.sh

```diff
@@ -120,6 +120,7 @@ if [ -z "${MAVEN_HOME}" ]; then
   export PATH="${MAVEN_HOME}/bin:${PATH}"
 fi
 export MAVEN_OPTS="${MAVEN_OPTS} -Dmaven.repo.local=${local_repo}"
+export MAVEN_ARGS="--threads=0.5C ${MAVEN_ARGS}"
```

Comment:
trying to optimize the website build at the same time as the precommit / nightly tests feels overloaded to me. How are we going to test this change?

Reply:
Was just trying to cover all the mvn references. Let me drop this for now. Thanks.